.circleci/config.yml

```diff
@@ -39,19 +39,23 @@ jobs:
           name: 'Creating Virtualenv'
           command: |
             pyenv virtualenv 3.9.4 venv
+      - restore_cache:
+          key:
+            sceptre-{{ .Environment.CACHE_VERSION }}-dependencies-{{ arch }}-{{
+            checksum "requirements/prod.txt" }}-{{ checksum
+            "requirements/dev.txt" }}
```

Comment:
This spans multiple lines weird... You sure this is valid yaml?

Reply:
It's been like this before I ever even touched this file.  I'm just going with the flow.  And yes, it works.